flexmeasures/api/v3_0/sensors.py

```diff
@@ -576,3 +577,56 @@ def post(self, sensor_data: dict):
         db.session.add(sensor)
         db.session.commit()
         return sensor_schema.dump(sensor), 201
+
+    @route("/<id>", methods=["PATCH"])
+    @use_args(partial_sensor_schema)
+    @use_kwargs({"db_sensor": SensorIdField(data_key="id")}, location="path")
+    @permission_required_for_context("update", ctx_arg_name="db_sensor")
+    @as_json
+    def patch(self, sensor_data: dict, id: int, db_sensor: Sensor):
+        """Update an sensor given its identifier.
+
+        .. :quickref: Sensor; Update a sensor
+
+        This endpoint sets data for an existing sensor.
+        Any subset of sensor fields can be sent.
+
+        The following fields are not allowed to be updated:
+        - id
+        - generic_asset_id
+
+        **Example request**
+
+        .. sourcecode:: json
+
+            {
+                "name": "POWER",
+            }
+
+        **Example response**
+
+        The whole sensor is returned in the response:
+
+        .. sourcecode:: json
+
+            {
```

Comment:
What about `timezone` and `entity_address`? Wouldn't they also be returned here?

Reply:
updated the documentation. All sensor fields are returned indeed